ArduPlane/events.cpp

```diff
@@ -301,7 +301,11 @@ void Plane::handle_battery_failsafe(const char *type_str, const int8_t action)
             }
             FALLTHROUGH;
         }
-        case Failsafe_Action_RTL: {
+        case Failsafe_Action_RTL:
+#if MODE_AUTOLAND_ENABLED
```

Comment:
I wounder if its correct to define out the action here. If someone sets it then we should RTL if auto land is not compiled in.

Reply:
I have pushed a commit that makes that change.